src/material-experimental/mdc-chips/chip.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
I would name this file `chip-cell.ts` since `MatChip` by itself with no role should still be a thing.